apps/vue-app/src/app/modules/posts/views/PostDetailView.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <div class="row">
+    <div class="col-md-12">
+      <div class="card bg-dark text-white">
+        <img src="https://cdn.pixabay.com/photo/2017/02/22/17/06/wave-2089959_960_720.jpg" class="card-img" />
+        <div class="card-img-overlay">
+          <div class="d-flex justify-content-start align-items-center ms-4" v-on:click="goBack()">
+            <i class="fa-solid fa-chevron-left me-2"></i>
+            <span>View Posts</span>
+          </div>
+        </div>
+        <div class="card-img-overlay text-center title">
+          <div class="card-content">
+            <h1 class="display-2">
+              <strong>{{ this.post.title }}</strong>
```

Comment:
Dentro del <template> no es necesario agregar this.* para accerder a la data de tu componente